[#81] [#82] [#83] 리팩토링 & 테스트 코드 작성

lacomaco opened the pull request:
- 전역으로 선언했던 newData 변수를 스코프 안으로 밀어넣는 리팩토링을 수행했고 오타를 몇개 찾아서 고쳤어요.
- 작성한 model의 reducer에 대한 테스트를 수행했습니다.
- 작성항 contextAPI Model에 대한 컴포넌트 테스트를 수행했습니다.

mon823 committed:
Merge pull request #80 from boostcamp-2020/dev

[WEEK1] demo

yyjjjj committed:
Update README.md

+ 팀 관련 기본사항 추가

EarlyHail committed:
[REFACTOR] 파일, 폴더 이름 수정
- 파일, 폴더 이름 case error 수정

lacomaco committed:
[REFACTOR] IssueListModel 오타 수정

- clonedeep을 cloneDeep으로 수정하였습니다.

lacomaco committed:
[REFACTOR] [#81] IssueList New Data 리팩토링

- 전역 변수로 선언된 NewData 변수를 스코프로 밀어넣는 리팩토링을
  수행했습니다.

lacomaco committed:
[TEST] [#82] IssueListModelReducer 테스트 코드

- IssueListInitialize에 대한 테스트 코드를 작성했습니다
- IssueCheckToggle에 대한 테스트 코드를 작성했습니다.
- IssueCheckAll에 대한 테스트 코드를 작성했습니다.
- IssueUnCheckAll에 대한 테스트 코드를 작성했습니다.

lacomaco committed:
[TEST] [#83] IssueListModel Context API 테스트 코드 작성

- IssueListModel Context API의 4개의 액션 함수에 대한 테스트 코드를
  작성했습니다.

- Dummy Component에 적용하여 실제로 액션 함수가 작동 하는지 컴포넌트
  테스트 코드를 작성하였습니다.

mon823 reviewed (approved):
늦은 시간에도 테스트코드까지 ㄷㄷ 다들 대단하시네요 ㅠㅠ 제가 바뻐서 8시반 넘어서 도움을 못드리네요 ㅠㅠ

EarlyHail commented on frontEnd/src/models/issueListModel.js:
```diff
@@ -52,36 +52,38 @@ export const IssueListModelContext = createContext();
+export function reducer(state, action) {
   switch (action.type) {
-    case IssueListInitialize:
+    case IssueListInitialize: {
```
범인은 `{}`였습니다.....

EarlyHail reviewed (approved):
파일, 폴더 네이밍을 어떻게 수정했는지 간단하게 써주시면 좋을것 같아요!!!
이따 스크럼 끝나고 이야기해보는걸로.....
고생하셨어요! 👍

yyjjjj reviewed (approved):
아아... const 관련 에러가 중괄호 때문이었군요.. 고생하셨습니다!!

lacomaco committed:
[REFACTOR] [#85] CSS 리팩토링

- IssueList의 Border에 light-gray를 추가하였습니다.
- 체크박스 열을 정렬하였습니다.
- 2중 border 문제를 해결했습니다.
- NewIssue Button이 뭉개지는 현상을 해결했습니다.
- Issue에 대한 전반적인 스타일 작업을 수행했습니다.
- Filter Button의 font-size를 16px으로 수정하였습니다.
